### ISSUE TITLE
Viafree - No videos found. Can't get rtmpparse info

### ISSUE THREAD

**Issue by TXC** [spaam/svtplay-dl#950]
Trying to download Steve Angelos live performance from Malmöfestivalen last night.

```
# svtplay-dl --verbose -o /mnt/disk3/Streams/ https://www.viafree.se/program/underhallning/malmofestivalen/sasong-2018/970826
version: 2.0+1.g8297d66
DEBUG [1534589968.0922167] /usr/local/lib/python3.6/dist-packages/svtplay_dl-2.0+1.g8297d66-py3.6.egg/svtplay_dl/utils/getmedia.py/get_media: version: 2.0+1.g8297d66
DEBUG [1534589968.0935597] /usr/local/lib/python3.6/dist-packages/svtplay_dl-2.0+1.g8297d66-py3.6.egg/svtplay_dl/service/__init__.py/__init__: service: viaplay
DEBUG [1534589968.0936718] /usr/local/lib/python3.6/dist-packages/svtplay_dl-2.0+1.g8297d66-py3.6.egg/svtplay_dl/utils/http.py/request: HTTP getting 'https://www.viafree.se/program/underhallning/malmofestivalen/sasong-2018/970826'
DEBUG [1534589968.095113] /usr/local/lib/python3.6/dist-packages/urllib3-1.23-py3.6.egg/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): www.viafree.se:443
DEBUG [1534589968.1572125] /usr/local/lib/python3.6/dist-packages/urllib3-1.23-py3.6.egg/urllib3/connectionpool.py/_make_request: https://www.viafree.se:443 "GET /program/underhallning/malmofestivalen/sasong-2018/970826 HTTP/1.1" 200 None
DEBUG [1534589968.1914783] /usr/local/lib/python3.6/dist-packages/svtplay_dl-2.0+1.g8297d66-py3.6.egg/svtplay_dl/utils/http.py/request: HTTP getting 'http://playapi.mtgx.tv/v3/videos/970826'
DEBUG [1534589968.1926024] /usr/local/lib/python3.6/dist-packages/urllib3-1.23-py3.6.egg/urllib3/connectionpool.py/_new_conn: Starting new HTTP connection (1): playapi.mtgx.tv:80
DEBUG [1534589968.2304149] /usr/local/lib/python3.6/dist-packages/urllib3-1.23-py3.6.egg/urllib3/connectionpool.py/_make_request: http://playapi.mtgx.tv:80 "GET /v3/videos/970826 HTTP/1.1" 200 2084
DEBUG [1534589968.2313282] /usr/local/lib/python3.6/dist-packages/svtplay_dl-2.0+1.g8297d66-py3.6.egg/svtplay_dl/utils/http.py/request: HTTP getting 'http://playapi.mtgx.tv/v3/videos/stream/970826'
DEBUG [1534589968.2942386] /usr/local/lib/python3.6/dist-packages/urllib3-1.23-py3.6.egg/urllib3/connectionpool.py/_make_request: http://playapi.mtgx.tv:80 "GET /v3/videos/stream/970826 HTTP/1.1" 200 378
ERROR [1534589968.2951052] /usr/local/lib/python3.6/dist-packages/svtplay_dl-2.0+1.g8297d66-py3.6.egg/svtplay_dl/utils/getmedia.py/get_one_media: No videos found. Can't get rtmpparse info
```

**Comment by Sopor-** [comment]
I get the same for https://www.viafree.se/program/samhalle/efterlyst/sasong-52/972302

**Comment by spaam** [comment]
hmmm... i cant reproduce it right now. i guess i have to wait for tomorrow when efterlyst is live again

**Comment by Sopor-** [comment]
It seems that it take some time before they change it from live stream to a regular stream even after the program has ended.

**Comment by Sopor-** [comment]
@spaam Now you have a few hours before [Efterlyst](https://www.viafree.se/program/samhalle/efterlyst) change :)

**Comment by spaam** [comment]
this will be fixed in next release 2.1 tomorrow friday.  thanks for reporting <3
